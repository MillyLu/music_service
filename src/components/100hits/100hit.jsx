
/* eslint-disable react/jsx-no-bind */

import { useContext} from 'react';

import  Navigation  from "../navigation/nav";
import { ThemeContext } from '../../ThemeProvider';
import * as Styled from './styles'

import { Footer } from '../footer/footer';
import { HitsCenterBlock } from '../100hits-centerBlock/hitsCenterBlock';



/* function HitsTitle() {
    const {theme} = useContext(ThemeContext);
    return(
        <Styled.CenterblockTitle theme={theme}>100 Танцевальных хитов</Styled.CenterblockTitle>
    )
} */




/* function Footer() {
    return(
        <footer className='footer' />
    )
} */

/* function Main() {
    const {theme} = useContext(ThemeContext);
    return(
        <Styled.Main theme={theme}>
            <Navigation />
            <HitsCenterBlock />
        </Styled.Main>
    )
} */

export function PlaylistContainer() {
    const {theme} = useContext(ThemeContext);
    return(
        <Styled.PlaylistContainer theme={theme}>
            <Styled.Main theme={theme}>
                <Navigation />
                <HitsCenterBlock />
            </Styled.Main>
            <Footer />
        </Styled.PlaylistContainer>
    )
}