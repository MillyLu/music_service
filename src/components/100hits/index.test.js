/* eslint-disable no-undef */
import * as ShallowRenderer from 'react-shallow-renderer';
// import { ShallowRenderer, createRenderer } from 'react-shallow-renderer';
import { PlaylistContainer } from "./100hit";



describe('<PlaylistContainer/>', () => {
        it('should render successfully and match snapshot', () => {
            const shallowRenderer = ShallowRenderer.createRenderer();
            const renderer = shallowRenderer;
            renderer.render(<PlaylistContainer />);
            const result = renderer.getRenderOutput();
            expect(result).toMatchSnapshot();
          })
      });

