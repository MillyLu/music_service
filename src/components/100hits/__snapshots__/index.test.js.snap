// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlaylistContainer/> should render successfully and match snapshot 1`] = `
<styled.div
  theme={0}
>
  <styled.div
    theme={0}
  >
    <Navigation />
    <HitsCenterBlock />
  </styled.div>
  <Footer />
</styled.div>
`;
